exporters/stats/prometheus/src/test/java/io/opencensus/exporter/stats/prometheus/PrometheusExportUtilsTest.java

```diff
@@ -188,22 +188,49 @@ public void getType() {
   public void createDescribableMetricFamilySamples() {
     assertThat(
             PrometheusExportUtils.createDescribableMetricFamilySamples(
-                CUMULATIVE_METRIC_DESCRIPTOR))
+                CUMULATIVE_METRIC_DESCRIPTOR, ""))
         .isEqualTo(
             new MetricFamilySamples(
                 METRIC_NAME, Type.COUNTER, METRIC_DESCRIPTION, Collections.<Sample>emptyList()));
     assertThat(
-            PrometheusExportUtils.createDescribableMetricFamilySamples(SUMMARY_METRIC_DESCRIPTOR))
+            PrometheusExportUtils.createDescribableMetricFamilySamples(
+                SUMMARY_METRIC_DESCRIPTOR, ""))
         .isEqualTo(
             new MetricFamilySamples(
                 METRIC_NAME2, Type.SUMMARY, METRIC_DESCRIPTION, Collections.<Sample>emptyList()));
     assertThat(
-            PrometheusExportUtils.createDescribableMetricFamilySamples(HISTOGRAM_METRIC_DESCRIPTOR))
+            PrometheusExportUtils.createDescribableMetricFamilySamples(
+                HISTOGRAM_METRIC_DESCRIPTOR, ""))
         .isEqualTo(
             new MetricFamilySamples(
                 METRIC_NAME3, Type.HISTOGRAM, METRIC_DESCRIPTION, Collections.<Sample>emptyList()));
   }
 
+  @Test
+  public void createDescribableMetricFamilySamples_WithNamespace() {
+    String namespace1 = "myorg";
+    assertThat(
+            PrometheusExportUtils.createDescribableMetricFamilySamples(
+                CUMULATIVE_METRIC_DESCRIPTOR, namespace1))
+        .isEqualTo(
+            new MetricFamilySamples(
+                namespace1 + '_' + METRIC_NAME,
+                Type.COUNTER,
+                METRIC_DESCRIPTION,
+                Collections.<Sample>emptyList()));
+
+    String namespace2 = "opencensus/";
+    assertThat(
+            PrometheusExportUtils.createDescribableMetricFamilySamples(
+                CUMULATIVE_METRIC_DESCRIPTOR, namespace2))
+        .isEqualTo(
+            new MetricFamilySamples(
+                "opencensus_" + METRIC_NAME,
```

Comment:
Should this be "opencensus/" + METRIC_NAME

Reply:
No because Promtheus exporter automatically sanitizes non-alphanumeric characters to underscores. See https://prometheus.io/docs/concepts/data_model/#metric-names-and-labels: 

> It may contain ASCII letters and digits, as well as underscores and colons. It must match the regex [a-zA-Z_:][a-zA-Z0-9_:]*.